server/oneoff/sync_item.py

```diff
@@ -9,8 +9,8 @@
 from app.database.model_item_type_translation import ModelItemTypeTranslation
 from app.database.model_weapon_effect import ModelWeaponEffect
 from app.database.model_weapon_stat import ModelWeaponStat
-from oneoff.database_setup import to_stat_enum
-from oneoff.database_setup import to_effect_enum
+import oneoff.database_setup
+import oneoff.database_setup
```

Comment:
What's the motivation here? I think I prefer the way it was before, since you don't need all of database_setup. Also, I think we may need to move those enums here and then import them from here to database_setup to avoid circular dependencies.